plottable.d.ts

```diff
@@ -1449,7 +1449,7 @@ declare module Plottable {
          * @param {String|d3.Selection} element A selector-string for the <svg>, or a d3 selection containing an <svg>.
          * @returns {Component} The calling Component.
          */
-        renderTo(element: String | d3.Selection<void>): Component;
+        renderTo(element: String | Element | d3.Selection<void>): Component;
```

Comment:
I learned something today. SVGSVGElement are not HTMLElement, but they are Element. Cool